docs/Architecture Decision Record/010-buildpacks.md

```diff
@@ -0,0 +1,78 @@
+# 1. Record architecture decisions
```

Comment:
There is already a ADR #10 in the pipeline, so this will have to be changed to 11.

Also, this line should be updated as follows
# 11. Cloud.gov Buildpacks